lib/components/narrative/tabbed-itineraries.js

```diff
@@ -105,54 +111,69 @@ class TabButton extends Component {
   }
 
   render () {
-    const {index, isActive, itinerary, timeFormat} = this.props
-    const timeOptions = {
-      format: timeFormat,
-      offset: getTimeZoneOffset(itinerary)
-    }
+    const {currency, index, isActive, itinerary, timeFormat} = this.props
     const classNames = ['tab-button', 'clear-button-formatting']
     const { caloriesBurned } = calculatePhysicalActivity(itinerary)
     const {
-      centsToString,
       maxTNCFare,
       minTNCFare,
       transitFare
     } = calculateFares(itinerary)
     // TODO: support non-USD
     const minTotalFare = minTNCFare * 100 + transitFare
-    const plus = maxTNCFare && maxTNCFare > minTNCFare ? '+' : ''
     if (isActive) classNames.push('selected')
     return (
       <Button
         className={classNames.join(' ')}
         key={`tab-button-${index}`}
         onClick={this._onClick}
       >
-        <div className='title'><span>Option {index + 1}</span></div>
+        <div className='title'><span>
+          <FormattedMessage
+            id='components.TabbedItineraries.optionNumber'
+            values={{optionNum: index + 1}}
+          />
+        </span></div>
```

Comment:
Formatting suggestion:

```suggestion
        <div className='title'>
          <span>
            <FormattedMessage
              id='components.TabbedItineraries.optionNumber'
              values={{optionNum: index + 1}}
            />
          </span>
        </div>
```

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84